lib/charms/tempo_k8s/v2/tracing.py

```diff
@@ -118,10 +118,6 @@ def __init__(self, *args):
 
 ReceiverProtocol = Literal[
     "zipkin",
-    "kafka",
-    "opencensus",
-    "tempo_http",
-    "tempo_grpc",
     "otlp_grpc",
```

Comment:
The protocols aren't needed in `receiver_protocol_to_transport_protocol` either:

https://github.com/canonical/tempo-k8s-operator/pull/149/files#diff-b47b4334954f653c5e1d7a8831211b30280d18fa54a07681f9108cec9ce34923R140 

(where I somehow can't leave a comment)

Reply:
ehm, funny enough, it seems that Tempo is supporting jaeger thrift

Reply:
can go either way with it now. Do we add it to the supported protocols or strip it completely?

Reply:
ah but wait, we don't really care. This is monolithic tempo. IIRC we've removed it already in the distributed tempo?

Reply:
nope, still there.

Reply:
It's in an uncertain state at the moment: https://github.com/canonical/tempo-coordinator-k8s-operator/blob/main/src/tempo.py#L53 still has it. I can remove it together with nginx PR if you think it's the way to go. 
@michaeldmitry, any opinions here maybe?

Reply:
My understanding is that we're going to drop Jaeger support at some point? https://www.jaegertracing.io/docs/1.58/apis/#opentelemetry-protocol-stable
That's why I didn't expose it as a config option, but looks like I missed it in https://github.com/canonical/tempo-coordinator-k8s-operator/blob/main/src/tempo.py#L53.

Reply:
Continuing in https://github.com/canonical/tempo-coordinator-k8s-operator/issues/8